app/routes/photos/components/GalleryEditorActions.js

```diff
@@ -26,37 +26,31 @@ const GalleryEditorActions = ({
 }: Props) => (
   <Collapse isOpened={selectedCount > 0}>
     <Sticky enabled={selectedCount > 0} innerZ={10} top={0}>
-      <div className={styles.actionsContainer}>
-        <Flex className={styles.actions} justifyContent="space-between">
-          <span>
-            <Icon
-              className={styles.deselectIcon}
-              onClick={onDeselect}
-              name="close"
-            />
-            {selectedCount} Selected
-          </span>
-          <div className={styles.actionBar}>
+      <Flex className={styles.actionsContainer}>
+        <Flex className={styles.actionBar}>
```

Comment:
Does this do anyting? I.e the inner `Flex` (actionBar) does not have any flex properties. I think you need a `FlexItem` between here, but I'm not 100% sure though.

Reply:
+1 The double flex will have an effect, due to the classNames used, but they will override each other.

I recommend that you remove the inner Flex (actionBar),  and delete that css-class, then make some changes to the actionContainer instead

Then make the required changes to `actionContainer` to make it look how you like.

Changing the justify-content to space-between will make it look almoast the same. Maybe change some padding aswell
```sh
justify-content: space-between;
```